docs/conductor/getting-started.md

```diff
@@ -0,0 +1,93 @@
+# Getting started with Conductor
+##
+
+Conductor will group and schedule automated Dependency Updates on your own Continuous Integration platform. If the update succeeds, Conductor sends you a pull request to your code hosting platform (GitHub, GitLab, Bitbucket) with the changed composer.lock file and, if necessary, other files modified by Composer plugins or scripts.
+
+To use Conductor:
+
+- You need to be approved for early access to conductor. [Join to the waitlist](http://packagist.com.lo/features/conductor) and wait for approval.
+- You need a Private Packagist trial or subscription (Cloud or Self-Hosted).
+- You need to add a workflow to your Continuous Integration platform to run Composer updates, described below.
+
+## First steps
+
+Once you have a Private Packagist Subscription and Conductor is enabled for your subscription, log into your Private Packagist organization and click on the "Updates" tab in the main navigation.  
+Conductor will list packages:
+ - added to Private Packagist via synchronization
+ - having a composer.lock file commited to the repository
+ - having the Private Packagist repository added to the composer.json
+
+If you do not see your package, follow the instructions on your organization page to add the custom repository to the composer.json of your package.  
+
+Most convenient way to set up Conductor is to configure your CI in the Private Packagist UI. The manual steps are outlined below:
+
+## Create a workflow on your CI
+
+### GitHub Actions
+
+Create a new GitHub Actions workflow in `.github/workflows/dependency-update.yaml` of your package repository using the template below:
+
+CONDUCTOR_GITHUB_ACTIONS_WORKFLOW
+
+1. Adjust the PHP Version used in the "Install PHP" step
+2. Commit and push the workflow to your main branch of your package repository
+3. Review the steps to commit and push to your repository
+
+Create a secret `COMPOSER_AUTH` with the Composer authentication configuration [as described here](https://getcomposer.org/doc/articles/authentication-for-private-packages.md#authentication-using-the-composer-auth-environment-variable) to access Private Packagist.  
+We recommend to create a dedicated authentication token with update access. You can copy and paste the contents for the secret from the Private Packagist UI while creating the token in "Settings" -> "Authentication Tokens". Remove the single quotes around the value.
+
+![Create Authentication Token](/Resources/public/img/docs/conductor/authentication-token.png)
+
+The contents of the variable should look like 
+
+```json
+{"http-basic": {"repo.packagist.com": {"username": "token", "password": "packagist_out_73a81c..." }}}
+```
+
+Conductor needs to verify your setup before you can [start receiving Pull Requests](#how-scheduling-works). 
+
+## How scheduling works
+
+- Go to your package on the updates tab in your Private Packagist organization 
+- Click on the name of your package
```

Comment:
```suggestion
- Navigate to the updates tab in your Private Packagist organization 
- Click on the name of your package
```
If we need to specify what to click, maybe the UI isn't clear enough but couldn't really come up with a better verb/selector